src/integration-test/java/com/mongodb/jdbc/integration/MongoSQLIntegrationTest.java

```diff
@@ -55,4 +77,70 @@ public void loadTestConfigs() throws IOException {
         }
         return dynamicTests;
     }
+
+    @Test
+    public void testTypesOther() throws SQLException, IOException, IllegalAccessException {
+        DataLoader loader = new DataLoader(TEST_DATA_OTHER_DIRECTORY);
+        loader.dropCollections();
+
+        Map<String, Object> row = new LinkedHashMap<>();
+        ObjectId objID = new ObjectId("000000000000000000000001");
+        BsonObjectId objectID = new BsonObjectId(objID);
+        BsonDbPointer dbPointer = new BsonDbPointer("namespace", objID);
+        BsonJavaScript javaScript = new BsonJavaScript("function(){ }");
+        BsonRegularExpression regularExpression = new BsonRegularExpression("a(bc)*");
+        BsonSymbol symbol = new BsonSymbol("symbol");
+        BsonTimestamp timestamp = new BsonTimestamp(100);
+
+        row.put("_id", objectID);
+        row.put("a", dbPointer);
```

Comment:
perhaps more useful names could make this easier at a glance? "dbPointer", "javaScript", and so on